src/pages/schedule/schedule.ts

```diff
@@ -85,7 +83,10 @@ export class SchedulePage {
   goToSessionDetail(sessionData) {
     // go to the session detail page
     // and pass in the session data
-    this.navCtrl.push(SessionDetailPage, sessionData);
+    this.navCtrl.push(SessionDetailPage, {
+      data: sessionData,
+      session: sessionData.name
```

Comment:
Why are we passing name in a separate property? This is already passed as part of sessionData.